### PR TITLE
CON-7849 / Extract annotation parsing on service objects into the DO package

### PR DESCRIPTION
**Pull request by elijahomolo**
https://jira.internal.digitalocean.com/browse/CON-7849

### CODE REVIEW

**Review by timoreimann — CHANGES_REQUESTED** [review]
The annotations were moved to a different file only without adding a new package. Go-wise, this requires creating a new directory (like `common`) that should probably live in the root so that it can be referenced by both the CCM implementation and the future web hook.

I'd also appreciate if we could add a proper PR description. (I think it's fine to share publicly what we are aiming for.) Bonus points for adding a public, tracking Github issue that outlines the goal / feature.

File: cloud-controller-manager/do/lb_annotations.go [hunk 1]
```diff
@@ -0,0 +1,167 @@
+/*
+Copyright 2020 DigitalOcean
```
Comment (timoreimann) [inline cloud-controller-manager/do/lb_annotations.go:2]: Could use the opportunity to update to 2022.

**Review by timoreimann — APPROVED** [review]
LGTM with one optional suggestion.

File: cloud-controller-manager/do/lb_annotations.go (hunk 1 above)
Comment (cpanato) [reply cloud-controller-manager/do/lb_annotations.go:2]: Would say 2023 :)

Comment (timoreimann) [reply cloud-controller-manager/do/lb_annotations.go:2]: Haha yeah good catch. I'm still stuck in the past it seems. 😀

**Pull request merged**